charts/redpanda/chart.go

```diff
@@ -109,7 +114,195 @@ func Template(release helmette.Release, values PartialValues, kubeConfig kube.Co
 		j++
 	}
 
-	return manifests[:j], nil
+	manifests = manifests[:j]
+
+	v := helmette.UnmarshalInto[Values](dot.Values)
```

Comment:
This should be extracted into it's own function. I think we'd benefit greatly from making this code run through gotohelm so we can finally put the `console/configmap-and-deployment.yaml` to rest. We'll need to do the same for `connectors` any ways and I have no desire to maintain both this block and the helm file.

A hacky but possibly quick solution could be updating gotohelm to accept a list of packages as "subcharts" which don't get transpiled but function references get transpiled as if they were. ie:

```go
console.Deployment(...)
```

Would become (with the correct calling convention of course):

```
{{ "console.Deployment" ... }}
```

and we'd assume that `console.Deployment` is available due to subcharting.